universal-login-commons/lib/core/utils/sanitizers/asHexString.ts

```diff
@@ -0,0 +1,13 @@
+import {utils} from 'ethers';
+import {Sanitizer, Result} from '@restless/sanitizers';
+
+export const asHexString: (length: number) => Sanitizer<string> = (length: number) => {
+  return (value: unknown, path: string) => {
+    if (typeof value === 'string' && utils.isHexString(value) && value.length === length) {
```

Comment:
Make consistent with `isProperHexString(..)`
https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-commons/lib/core/utils/hexStrings.ts

Reply:
We should delete our own `isProperHexString` implementation and use the one in etheres